### ISSUE TITLE
Introduce labels to specify what journeys to run

### ISSUE THREAD

**Issue by jakobgrunig** [hmrc/performance-test-runner#7]
I would like to be able to use a list of labels to tell if a journey has to run or not

For example:

I have 2 different load I would like to test. One expected on week days and a different one on weekends.
I would like to be able to tell if a journey is intended to run on weekends only. Then, when I start the test I would like to use a lable to trigger one or the other test.

Example:
```
my-week-days-journey {
    description = "Week days journey"
    load = 6.2
    feeder = data/helloworld.csv
    parts = [
      login
    ],
    run-if = [sunday]
}

my-weekend-journey {
    description = "Weekend journey"
    load = 3.8
    feeder = data/helloworld.csv
    parts = [
      login
    ],
    skip-if = [sunday]
}
```

1. Introduce a new `perftest` configuration keys to specify a list of labels: 'perftest.label'
2. Introduce two new `journeys` configuration keys to specify if a journey should be run or not:
```
run-if = [sunday]
skip-if = [sunday]
```

If one of the labels in `perftest.labels` is present in `skip-if`, skip the journey
If one of the labels in `perftest.labels` is present in `run-if`, run the journey
In case of conflicts (labels are present in both), throw an exception

**Comment by paoloboni** [comment]
Closing as PR https://github.com/hmrc/performance-test-runner/pull/10 has been merged.

**Comment by jakobgrunig** [comment]
Released in performance-test-runner 1.10.0
